### PR TITLE
[32786] Can not switch between work packages in split view

### PR DESCRIPTION
**Pull request by HDinger**
Allow switch of details view on all tabs not only on overview.

https://community.openproject.com/projects/openproject/work_packages/32786/activity

### CODE REVIEW

File: frontend/src/app/components/wp-fast-table/handlers/row/click-handler.ts [hunk 1]
```diff
@@ -62,11 +62,6 @@ export class RowClickHandler implements TableEventHandler {
       return true;
     }
 
-    // Ignore links
-    if (target.is('a') || target.parent().is('a')) {
```
Comment (oliverguenther) [inline frontend/src/app/components/wp-fast-table/handlers/row/click-handler.ts:66]: Why did you remove this? I remember to added this because the link itself has a `uiSref` that should open the split view. And if you click it with a modifier to open in new window or tab, this line will prevent opening the details in the old window/tab

Comment (HDinger) [reply frontend/src/app/components/wp-fast-table/handlers/row/click-handler.ts:66]: Because exactly the same code is already written a few lines before: https://github.com/opf/openproject/blob/bf86eef503a37da30138e6cef0fb0273c75c6cf8/frontend/src/app/components/wp-fast-table/handlers/row/click-handler.ts#L45

Comment (oliverguenther) [reply frontend/src/app/components/wp-fast-table/handlers/row/click-handler.ts:66]: That answer I can accept :laughing:

**Review by oliverguenther — APPROVED** [review]
